logging.go

```diff
@@ -292,6 +292,9 @@ type BaseLog struct {
 	// The encoder is how the log entries are formatted or encoded.
 	EncoderRaw json.RawMessage `json:"encoder,omitempty" caddy:"namespace=caddy.logging.encoders inline_key=format"`
 
+	// Tees log entries through a zap.Core module
```

Comment:
I think this could use a deeper explanation, similarly because it shows up in JSON docs. i.e. I think it would be good if here it said "what can you do with a Core that can't be done otherwise"